executor/seqtest/prepared_serial_test.go

```diff
@@ -819,57 +856,59 @@ func (msm *mockSessionManager1) ShowProcessList() map[uint64]*util.ProcessInfo {
 	return ret
 }
 
-func (msm *mockSessionManager1) GetProcessInfo(id uint64) (*util.ProcessInfo, bool) {
+func (msm *mockSessionManager1) GetProcessInfo(_ uint64) (*util.ProcessInfo, bool) {
 	pi := msm.Se.ShowProcess()
 	return pi, true
 }
 
 // Kill implements the SessionManager.Kill interface.
-func (msm *mockSessionManager1) Kill(cid uint64, query bool) {}
+func (msm *mockSessionManager1) Kill(_ uint64, _ bool) {}
 
 func (msm *mockSessionManager1) KillAllConnections() {}
 
 func (msm *mockSessionManager1) ServerID() uint64 {
 	return 1
 }
 
-func (msm *mockSessionManager1) UpdateTLSConfig(cfg *tls.Config) {}
+func (msm *mockSessionManager1) UpdateTLSConfig(_ *tls.Config) {}
 
-func (s *seqTestSuite) TestPreparedIssue17419(c *C) {
+func TestPreparedIssue17419(t *testing.T) {
+	store, dom, clean := testkit.CreateMockStoreAndDomain(t)
+	defer clean()
 	ctx := context.Background()
-	tk := testkit.NewTestKit(c, s.store)
+	tk := testkit.NewTestKit(t, store)
 
 	tk.MustExec("use test")
 	tk.MustExec("drop table if exists t")
 	tk.MustExec("create table t (a int)")
 	tk.MustExec("insert into t (a) values (1), (2), (3)")
 
-	tk1 := testkit.NewTestKit(c, s.store)
+	tk1 := testkit.NewTestKit(t, store)
 
-	var err error
-	tk1.Se, err = session.CreateSession4Test(s.store)
-	c.Assert(err, IsNil)
-	tk1.GetConnectionID()
+	se, err := session.CreateSession4Test(store)
+	require.NoError(t, err)
+	tk1.SetSession(se)
```

Comment:
The old `util/testkit.NewTestKit` will not create a session for testkit, but the new one `testkit.NewTestKit` will. Should we need to set again?

Reply:
Removed.